src/js/sdk.tsx

```diff
@@ -41,6 +41,7 @@ const DEFAULT_OPTIONS: ReactNativeOptions = {
   transportOptions: {
     textEncoder: makeUtf8TextEncoder(),
   },
+  sendClientReports: true,
```

Comment:
Is it disabled by default on JS?

Reply:
no

Reply:
So then this line does nothing?

Reply:
actually, we are not using the js default so it has to be there.
I got confused for a moment